README.md

```diff
@@ -1141,6 +1142,11 @@ In either case:
       ]
 
       # good
```

Comment:
is this block weird to read since it's all inside the same `class` and we're redeclaring (and reassigning) the same constants? what if did `RED`as bad and `GREEN` as good or something like that? or alternately, use two different `class Color` blocks and do the `bad` / `good` declaration out there?